### PR TITLE
Bananium Plated Clown Shoes

### PR DESCRIPTION
**Pull request by xxalpha**
You grab a pair of clown shoes, the yellowest banana ever grown and a good amount of bananium, you put it all on a rune and what do you get?

Nothing, you need to honk it first.

clwonruneshoes

### CODE REVIEW

**Comment by Xhuis** [comment]
yes

**Comment by Fayrik** [comment]
>Plays the adminhelp bwoink when emagged.

This is clowning done right. :+1:

**Comment by francinum** [comment]
:+1::+1::+1::+1::+1::+1:

**Comment by babinsquared** [comment]
oh my god yes

File: code/modules/clothing/shoes/bananashoes.dm [hunk 1]
```diff
@@ -58,4 +58,89 @@
...
+		if("Laser sword")
+			sound = "sound/weapons/blade1.ogg"
+		if("Gunshot")
+			sound = "sound/weapons/Gunshot[pick(2,3,4)].ogg"
+		if("Laser")
+			sound = "sound/weapons/Laser.ogg"
+		if("Taser")
+			sound = "sound/weapons/Taser.ogg"
+		if("%#$HELP")
+			sound = "sound/effects/adminhelp.ogg"
```
Comment (Miauw62) [inline code/modules/clothing/shoes/bananashoes.dm]: no

File: code/modules/clothing/shoes/bananashoes.dm [hunk 2]
```diff
@@ -58,4 +58,89 @@
...
+/obj/item/clothing/shoes/clown_shoes/bananium/step_action()
+	if(footstep > 1)
+		playsound(src, sound, 75, 1)
+		footstep = 0
+	else
+		footstep++
+
+/obj/item/clothing/shoes/clown_shoes/bananium/emag_act()
+	if(!emagged)
+		sound_list |= "%#$HELP"
```
Comment (Miauw62) [inline code/modules/clothing/shoes/bananashoes.dm]: no

**Comment by Miauw62** [comment]
no adminbwoink in player sounds.

File: code/modules/clothing/shoes/bananashoes.dm (hunk 2 above)
Comment (Dorsisdwarf) [inline code/modules/clothing/shoes/bananashoes.dm]: Says someone who's never encountered the banhammah toy.

File: code/modules/clothing/shoes/bananashoes.dm (hunk 1 above)
Comment (Dorsisdwarf) [inline code/modules/clothing/shoes/bananashoes.dm]: yes

File: code/modules/clothing/shoes/bananashoes.dm (hunk 2 above)
Comment (Miauw62) [inline code/modules/clothing/shoes/bananashoes.dm]: good thing the banhammer is literally adminspawn only.

**Comment by RemieRichards** [comment]
Only if the shoes make them Valid.

I'm not listening to this shit 24/7, the default is barely tolerable as it is.

**Comment by Kromgar** [comment]
Hearing bwoinks when the clown walks? JESUS GOD KILL ME NOW.

That or kill the clown

**Comment by xxalpha** [comment]
Not to deny the dangers of clown bwoinking, but the sound is played with a different pitch every time, so you could tell after a first scare that it's the clown shoes.

**Comment by RemieRichards** [comment]
That means fuck all.
1. Still an Admin only sound (for bwoinks)
2. Still an annoying as fuck effect.

**Comment by babinsquared** [comment]
Make it harder to get maybe. I love it when clowns go the extra mile to be funnoying.

**Comment by Miauw62** [comment]
Remove the adminbwoink sound.

**Comment by RemieRichards** [comment]
(just remove the idea tbh)

**Comment by ACCount12** [comment]
How about reducing slowdown for bananium shoes?

**Comment by ACCount12** [comment]
Why?

**Pull request closed without merging**